### PR TITLE
added smaller call-out to discuss on blog page

### LINKED ISSUE #2096: [Callout] Make a smaller callout for the blog page
**Current Behavior**
The newly designed callout for discuss doesn't sit well on the blog page.

**Desired Situation**
Create a smaller callout which sits well with the already existing callout on the blog page.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabh1908@gmail.com>

**Description**

This PR fixes #2096 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 2fd630723e3b322d0a78f0040b045349edd800a9 at: https://6120fd35a24365e567e4c3af--layer5ng.netlify.app

**Review by Jashpatel1 — COMMENTED** [review]
@debo19 It looks good. Just a small change, can you reduce the line 'Ask questions, find answers and share knowledge on our Discussion Forum' to something short, if possible. If not, remove the line.

**Comment by debo19** [comment]
@Jashpatel1 Should I just write :
"JOIN THE CONVERSATION 
  On our Discussion Forum" ? 
If this sounds vague then I'll remove it.

File: src/sections/Blog/Blog-sidebar/blogSidebar.style.js [hunk 1]
```diff
@@ -165,8 +165,54 @@ const BlogSideBarWrapper = styled.div`
...
+            max-width: 50rem;
+            padding: 1.5rem 1rem 0rem 1rem;
+            background-color: none;
+            border-radius: 25px;
+            .card {
+                -webkit-transition: 450ms all;
+                transition: 450ms all;
+                margin: 2px;
+                padding: 1.25rem;
+                background-color: #1E2117;
```
Comment (warunicorn19) [inline src/sections/Blog/Blog-sidebar/blogSidebar.style.js:182]: @debo19 could you please remove this hard-coded color. I think this is dark Jungle green, please use that from the themes.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Could you please align these together.
Also, is this a a different newsletter @Jashpatel1 ? 🤔 because we already have a newsletter subscription in our footer.
 
Screenshot 2021-08-21 at 7 11 24 PM

Also,
The callout disappears in the mobile view, if you could also fix that.

Screenshot 2021-08-21 at 7 11 24 PM

Overall it looks great, thanks @debo19

**Comment by debo19** [comment]
@warunicorn19 the newsletter subscription was hidden in this view previously that's why I didn't add the callout here too. I'll add it in now.

**Comment by l5io** [comment]
🚀 Preview for commit bf233b71ba6445223bc6a0c437f4c3b372853aa7 at: https://61212dbb37b15a18e3d6f888--layer5ng.netlify.app

**Review by Jashpatel1 — APPROVED** [review]

**Pull request merged**